Android/app/src/main/res/layout/activity_mmiw_ar.xml

```diff
@@ -53,6 +73,19 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent" />
 
+    <androidx.viewpager2.widget.ViewPager2
+        android:id="@+id/statisticsViewPager2"
```

Comment:
This is totally a nit but I feel like the fact that it is a ViewPager2 is an implementation detail that doesn't need to follow the naming of the view and it should just be statisticsViewPager. Mostly because when you see 2 in the title when you are using it in the views, it sounds like it is the second one and there is another viewpager on the screen.

Reply:
Haha, ok funny story. This was named because I was testing a few and this was the second one that worked. I went with this one, because it worked better. I didn't notice the connection. I'll update this :)